build-support/bin/native/bootstrap_rust.sh

```diff
@@ -23,6 +23,13 @@ function cargo_bin() {
   "${RUSTUP}" which cargo
 }
 
+function set_rust_toolchain() {
+    (
+      cd "${REPO_ROOT}"
+      "${RUSTUP}" override set "${RUST_TOOLCHAIN}" >&2
```

Comment:
Would it be more explicit to use the `rust-toolchain` file: https://github.com/rust-lang-nursery/rustup.rs#the-toolchain-file ?

Reply:
I think so. Good idea.